### PR TITLE
Bump the Android NDK version to r25.

### LINKED ISSUE #576: cloudbuild-ndk.yaml using NDK that hasn't been supported since 2018
## Affected builder image

https://github.com/GoogleCloudPlatform/cloud-builders-community/blob/master/android/cloudbuild-ndk.yaml

## Expected Behavior

Uses a supported NDK

## Actual Behavior

r17 wasn't even supported at the time this file was added

## Steps to Reproduce the Problem

1. Look at the file
2. See that it's using r17
3. Compare to the list of supported releases on https://github.com/android/ndk/wiki

## Additional Info

I haven't sent a PR updating the file because I'm not sure if the issue is the file choosing old images, or if there are no newer images to select.

### PR DESCRIPTION
**Pull request by khuey**
Closes #576

### CODE REVIEW

File: android/Dockerfile-base [hunk 1]
```diff
@@ -22,7 +22,7 @@ ENV LANG=C.UTF-8
 
 ARG sdk_version=sdk-tools-linux-4333796.zip
 ARG android_home=/opt/android/sdk
-ARG ndk_version=android-ndk-r17b
+ARG ndk_version=android-ndk-r23b
```
Comment (DanAlbert) [inline android/Dockerfile-base:25]: As of yesterday, these should be r25. That'll be in support for at least a year.

Comment (khuey) [reply android/Dockerfile-base]: Alright lets go to r25

Comment (DanAlbert) [reply android/Dockerfile-base]: Thanks!

**Review by sein2021 — APPROVED** [review]

**Review by sein2021 — APPROVED** [review]

**Comment by DanAlbert** [comment]
FYI in the time since this PR was uploaded NDK r25's support window has ended. r27 and r28 are the currently supported versions. If you stick to an LTS (currently r27) you'll only have to update this annually, but no NDK is supported longer than that.

**Pull request merged**